docs/tutorials/e2e/boost/consumeData.md

```diff
@@ -62,15 +66,39 @@ curl --location 'http://localhost/alice/management/v2/contractnegotiations' \
 }' | jq
 ```
 
-In the response, Alice gets a UUID. This is the ID of the created contract negotiation. Alice can now use this ID to see the current status of the negotiation and - if the negotiation was successful - the ID of the created contract agreement.
+The response should look like this:
+
+```json
+{
+  "@type": "edc:IdResponse",
+  "@id": "65356596-dd7c-4ad4-8fc6-8512be6f0ec2",
+  "edc:createdAt": 1715669329095,
+  "@context": {
+    "dct": "https://purl.org/dc/terms/",
```

Comment:
```suggestion
    "dct": "http://purl.org/dc/terms/",
```
This was faulty behavior in an EDC version way way back.

Reply:
we (@tunacicek and I) already adapted it in the last commit

Reply:
yea, sorry - haven't dug the versioning logic in github pr reviews yet.